HappyAnding/HappyAnding/Model/Alerter.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Alerter.swift
+//  HappyAnding
+//
+//  Created by 전지민 on 2023/02/07.
+//
+
+import SwiftUI
+
+class Alerter: ObservableObject {
+    @Published var showAlert = false
```

Comment:
```suggestion
    @Published var isPresented = false
```

대공사일 것 같지만.. Bool형이라면 변수 명 앞에 is를 붙이는 건 어떨까유

Reply:
우선 alerter의 showAlert 변수는 isPresented로 변경했습니다~! 제가 코드 컨벤션을 깜빡했는데 꼼꼼하게 챙겨주셔서 감사합니다!